### ISSUE TITLE
ShuffleNet difference with paper at first convolution

### ISSUE THREAD

**Issue by TheFlux7** [kuangliu/pytorch-cifar#35]
Hi @kuangliu,
Thank you for these great models!
For ShuffleNet, I noticed a difference between your implementation and the original paper. Before the ShuffleNet stages, you do a 1x1 convolution, followed by batch normalization and ReLU: `out = F.relu(self.bn1(self.conv1(x)))`, but in the original paper they perform a 3x3 convolution with stride 2 followed by a 3x3 maxpool with stride 2: 
![image](https://user-images.githubusercontent.com/38288098/38621130-7ad25068-3da0-11e8-9c23-cf699afb2ce9.png)
Was this done intentionally? Thank you in advance
TheFlux7

**Comment by YanShuo1992** [comment]
I believe the reason is the size of cifar images.  I see the similar change in mobilenet v2.
